netlify/functions/tweets.cjs

```diff
@@ -0,0 +1,29 @@
+const Twitter = require('twitter')
+require('dotenv').config()
+
+const client = new Twitter({
+  consumer_key: process.env.TWITTER_CONSUMER_KEY,
+  consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
+  bearer_token: process.env.TWITTER_BEARER_TOKEN,
+})
+
+const params = {
```

Comment:
**Note:** Some of these could be URL-parameterized. I'm curbing my urge to prematurely optimize 😅